plugins/cloud-azure/src/main/java/org/elasticsearch/cloud/azure/storage/AzureStorageService.java

```diff
@@ -44,9 +44,6 @@
 
         public static final String TIMEOUT = "cloud.azure.storage.timeout";
 
-        public static final String ACCOUNT = "repositories.azure.account";
-        public static final String LOCATION_MODE = "repositories.azure.location_mode";
-        public static final String KEY = "cloud.azure.storage.key";
         public static final String CONTAINER = "repositories.azure.container";
         public static final String BASE_PATH = "repositories.azure.base_path";
```

Comment:
It was just for integration tests actually. That way I was able to define this globally and run IT in their own random path name.
Let's remove it than. Thanks!